src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = new Random();
+    Color [] colors = Color.values();
```

Comment:
Please, add an access modifier here.

Reply:
Also, please note that your solution did not pass automatic verification. This is a check on the GitHub platform that validates your code for some errors. If you read the logs, you can see that she pointed out this mistake of yours.
<img width="907" alt="image" src="https://github.com/mate-academy/jv-lottery/assets/140437083/9a714939-b7e9-43ae-8594-523284578cb6">
<img width="1440" alt="image" src="https://github.com/mate-academy/jv-lottery/assets/140437083/ce7ad548-1255-4fff-a9c0-7e880a21ecd1">